CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ Mapbox welcomes participation and contributions from everyone.
       * Removing default values for parameters
       * Making `bearing` and `pitch` parameters optional
       * Adding the `camera(for:camera:rect:)` variant
+  - `OrnamentOptions` should now be accessed via `MapView.ornaments.options`. Updates can be applied directly to the `options` property. Previously the map's ornament options were updated via `MapConfig.ornaments`. ([#310](https://github.com/mapbox/mapbox-maps-ios/pull/310)) 
```

Comment:
So to be clear, the `mapView.update()` method no longer works for ornaments?

Reply:
Correct! I can update to make that more clear.